ovos_gui_plugin_shell_companion/brightness.py

```diff
@@ -75,10 +75,21 @@ def is_auto_dim_enabled(self, message=None):
 
     # Discover the brightness control device interface (HDMI / DSI) on the Raspberry PI
     def discover(self):
+        try:
+            vcgen_path = (subprocess.run(["which", "vcgencmd"], check=True,
```

Comment:
why not use `distutils.spawn.find_executable` like we do in most of our repos instead of a subprocess?

Reply:
`DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives`